cpp/include/cudf/legacy/quantiles.hpp

```diff
@@ -16,8 +16,8 @@
 
 #pragma once
 
-#include "cudf.h"
-#include "types.hpp"
+#include <cudf/cudf.h>
```

Comment:
Try to eliminate includes of cudf.h altogether. It just includes a whole lot of files. Change to only including the needed files.